pkgs/by-name/au/autotier/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  pkg-config,
+  rocksdb,
+  boost,
+  fuse3,
+  lib45d,
+  tbb_2021_11,
+  liburing,
+}:
+let
+  version = "1.2.0";
+in
+stdenv.mkDerivation {
+  name = "autotier";
+  version = version;
+  src = fetchFromGitHub {
+    owner = "45Drives";
+    repo = "autotier";
+    rev = "v${version}";
+    hash = "sha256-Pf1baDJsyt0ScASWrrgMu8+X5eZPGJSu0/LDQNHe1Ok=";
+  };
+
+  patches = [
+    # https://github.com/45Drives/autotier/pull/70
+    # fix "error: 'uintmax_t' has not been declared" build failure until next release
+    (fetchpatch {
+      url = "https://github.com/45Drives/autotier/commit/d447929dc4262f607d87cbc8ad40a54d64f5011a.patch";
+      hash = "sha256-0ab8YBgdJMxBHfOgUsgPpyUE1GyhAU3+WCYjYA2pjqo=";
+    })
+    # Unvendor rocksdb (nixpkgs already applies RTTI and PORTABLE flags) and use pkg-config for flags
+    (fetchpatch {
+      url = "https://github.com/45Drives/autotier/compare/master...philipwilk:autotier:unvendor.patch";
```

Comment:
This isn't reproducible enough. I think you should either fetch  each commit individually or add all commits as patches to Nixpkgs.